src/index.ts

```diff
@@ -66,6 +66,14 @@ const plugin = (options?: {uuid: string}): Plugin => ({
         root = path.resolve(process.cwd(), root);
       }
 
+      // WSL case detection
+      if (process.env.WSL_DISTRO_NAME) {
+        // Convert Linux path to Windows path format for WSL
+        root = path
+          .join("\\\\wsl.localhost", process.env.WSL_DISTRO_NAME, root)
```

Comment:
Doesn't `join` separate with `/` inside Linux container then?

Reply:
Yes, that's why I'm doing the replace after to change everything with `\`

Reply:
Ack